charts/astronomer/templates/houston/api/houston-deployment.yaml

```diff
@@ -71,6 +71,11 @@ spec:
             - name: SSLMODE
               value: {{ .Values.global.ssl.mode }}
             {{- end }}
+            # Connection Limit for Prisma Connection Pool: https://www.prisma.io/docs/concepts/components/prisma-client/deployment#database-connection-limit
+            {{ if .Values.global.connectionLimit.enabled }}
```

Comment:
This probably doesn't need to be a global setting. Maybe push the config down to just the `astronomer` chart values.yaml? Nested under the `houston` top-level key.